api/users.rb

```diff
@@ -19,6 +19,14 @@
   end
 end
 
+get "#{APIPREFIX}/users/:user_id/content_exists" do |user_id|
+  begin
+    Content.where(author_id: user_id).exists?.to_json
```

Comment:
Ruby is wild, yo. I have no idea if this is correct.

So the return is implied? And why the `to_json`? `exists?` is the function name, with the question mark?

This is a shockingly specific endpoint. Looking forward, like if we want to celebrate on the 100th post or something, we'd need a new endpoint? Wonder if an endpoint that returns the count of content items would be more flexible?

But it doesn't matter, really. I'm just pre-optimizing. If we need that stuff, we can refactor then.

Reply:
So that's actually a really easy change to make, so if we think it would be more flexible, I can update to count and switch endpoint to `.../content_count`. I went with `exists?` to start simply because I figured it would be more performant than count (although count seems to also be extremely fast).

And yeah, Ruby is cray. So from my understanding after spending some time in Ruby, yes, the last line in a function is the return value. The `to_json` is because it was raising an error without it and all the other endpoints do it so /shrug. The `exists?` is literally the function name (see https://docs.mongodb.com/mongoid/current/tutorials/mongoid-queries/ and then just ctrl+f for "exists")